Apps/PackageTest/0.63.1/package.json

```diff
@@ -10,7 +10,7 @@
     "lint": "eslint . --ext .js,.jsx,.ts,.tsx"
   },
   "dependencies": {
-    "@babylonjs/core": "5.2.0",
```

Comment:
Looks like the package-lock.json changes associated with this change are missing.